spec/models/rent_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Rent, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Si vamos a dejar un archivo que solo diga: acordate que acá tenemos que  agregar algunos test, me parece que el archivo en sí no tiene sentido.

Para que podamos dormir tranquilos de que hicimos las cosas bien :wink: :joy: solemos testear TODO lo que desarrollamos. En este caso, creamos un nuevo modelo `Rent` y deberíamos testear sus validaciones (por ejemplo validate_presence_of )

Nota general: siempre esta bueno ser consistentes con lo que hacemos: 
* Si hacemos una migracion -> Deberiamos correr `rails db:migrate` y ver el esquema modificado
* Si la migracion crea un nuevo modelo -> Deberíamos agregar un nuevo modelo y hacer las validaciones que pusimos en la migración (not nulls, has_one, belongs_to, etc)
* Si creamos un nuevo modelo -> Deberíamos hacer los test de ese nuevo modelo
* Para hacer los test del nuevo modelo -> Es posible que necesitemos una factory

:scream: son un montón de cosas, lo sé